.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+Checklist:
+
+* [ ] I have updated the necessary documentation
+* [ ] I have updated the changelog, if necessary (CHANGELOG.md)
```

Comment:
Should we have an "unreleased" section at the top of the changelog and ask people to specifically add their changes there? Given the lines below that say
> Maintainers will bump the version and do a release when they are ready to release (possibly multiple merged PRs). Please do not bump the version in your PRs.

If you don't have a version number or release date, making a changelog entry becomes a chicken and egg problem

Reply:
https://github.com/procore-oss/blueprinter#releasing-a-new-version

I'm not a fan of the unreleased idea.  I get the perspective though.  I'm good with removing the CHANGELOG idea.  I'd prefer that be automated by GitHub actions.  Wdyt?

Reply:
Removed but willing to put back if you feel strongly about it

Reply:
Removed seems fine. 
We loose the narrative of the developer who made the change
But looking back at included pull requests should allow whomever cuts a release to fill things out.
Just as long as cutting a release includes updating the changelog :-)

Reply:
I want to automate output like this

https://github.com/argoproj/argo-helm/releases/tag/argo-workflows-0.34.0

You see the list and the contributors

Reply:
Only possible really if we make [conventional git commits](https://www.conventionalcommits.org/en/v1.0.0/) required in the PR title.